### ISSUE TITLE
Wrong content section in the sidebar

### ISSUE THREAD

**Issue by mathbunnyru** [executablebooks/sphinx-book-theme#545]
### Describe the bug

**context**
I open <https://sphinx-book-theme.readthedocs.io/en/stable/tutorials/get-started.html#add-a-source-repository-button-to-your-theme> or press "Add a source repository button to your theme" content button on https://sphinx-book-theme.readthedocs.io/en/stable/tutorials/get-started.html.

**expectation**
I expected content section on the right panel to change to "Add a source repository button to your theme".

**bug**
But instead it tells me that the previous section is active.

If I scroll down a little bit, then the section is correct.

<img width="1552" alt="Снимок экрана 2022-03-29 в 17 14 56" src="https://user-images.githubusercontent.com/12270691/160658639-71a347f7-e3d8-4634-bfc2-da917886eec1.png">

### Reproduce the bug

1. Go to < https://sphinx-book-theme.readthedocs.io/en/stable/tutorials/get-started.html#add-a-source-repository-button-to-your-theme>

### List your environment

OS: macOS 12.3
Browser: Safari 15.4

**Comment by haklc** [comment]
I am noticing this too with v0.3.2. Have you found any way to resolve this @mathbunnyru?

**Comment by mathbunnyru** [comment]
No, I haven’t. But I didn’t actually try to find it, because web is definitely not my area of expertise.

**Comment by haklc** [comment]
It seems that the `active` class tag is being added to the wrong `<li>` entry in the right side menu (the entry above the actually active one).

Was this present in any of the previous versions? I just started using this theme.

**Comment by haklc** [comment]
Clicking on one of the entries in the menu positions you exactly 12 pixels above the chosen section.

**Comment by haklc** [comment]
Currently,  _data-offset_ parameter of the Bootstrap Scrollspy is set to "60" in the body tag of each HTML file. By changing this to "73" it works as expected, but there is some jittering. The optimal value seems to be "80".

**Comment by mathbunnyru** [comment]
> I just started using this theme.

Me as well, we've only started using it quite recently and the old versions don't work well for us for another reason.

@haklc could you make a PR changing this value of Bootstrap Scrollspy?

@choldgraf may I ask you to take a look here?

**Comment by choldgraf** [comment]
This is now fixed upstream, we just need to wait for a release then update here.

**Comment by mathbunnyru** [comment]
Great, thank you!

**Comment by haklc** [comment]
When is the next release expected? I'd love to use this theme but I cannot manually fix this value after HTML files are generated on Read the Docs.

**Comment by choldgraf** [comment]
There's a release candidate for the upstream theme out now (https://github.com/pydata/pydata-sphinx-theme/releases/tag/v0.9.0rc1) and then it's a question of doing the necessary changes to make this theme support it. No promises on timeline though.

**Comment by mathbunnyru** [comment]
The upstream new version has been released 🎉 
https://github.com/pydata/pydata-sphinx-theme/releases/tag/v0.9.0

**Comment by ClaudiaSchulz** [comment]
Any news as to when a newer version of pydata-sphinx-theme will be integrated into sphinx-book-theme?

**Comment by choldgraf** [comment]
Yep this will be closed by the following PR and we'll get a release candidate out soon after that:

- https://github.com/executablebooks/sphinx-book-theme/pull/640

**Comment by mathbunnyru** [comment]
I think this works much better now, but subtopic links are a bit broken.
https://jupyter-docker-stacks--1855.org.readthedocs.build/en/1855/#jupyter-notebook-deprecation-notice

When I visit this page, it doesn't show `Jupyter Notebook Deprecation Notice` subtopic (and I think it should).
<img width="1470" alt="Screenshot 2023-01-05 at 15 50 11" src="https://user-images.githubusercontent.com/12270691/210774431-05f001db-0931-4a56-9b0b-0f1656f0fec4.png">

I've used just released rc version: https://github.com/jupyter/docker-stacks/pull/1855

**Comment by choldgraf** [comment]
good catch - fixing it in:

- #669
